uint8 linhas_detectadas
float32[] angulos
float32[] altura_relativa
float32[] deslocamento_lateral
